v4/source/commands.yaml

```diff
@@ -166,6 +166,54 @@
 
             // ListAutocompleteCommands
             listCommands, resp := Client.ListAutocompleteCommands(<TEAMID>)
+  '/teams/{team_id}/commands/autocomplete_suggestions':
+    get:
+      tags:
+        - commands
+      summary: List commands' autocomplete data
+      description: |
+        List commands' autocomplete data for the team.
+        ##### Permissions
+        `view_team` for the team.
+      parameters:
+        - name: team_id
+          in: path
+          description: Team GUID
+          required: true
+          schema:
+            type: string
+        - name: user_input
+          in: query
+          description: String inputted by the user.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Commands' autocomplete data retrieval successful
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/AutocompleteSuggestion"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
```

Comment:
Out of curiosity: When does this happen?

Reply:
when you want to get commands for a team and you do not have a permission(for that team).

Reply:
Thanks for the explanation :+1: